Sources/TuistSupportTesting/Utils/MockGitHandler.swift

```diff
@@ -0,0 +1,27 @@
+import TSCBasic
+import TuistSupport
+
+final public class MockGitHandler: GitHandling {
+    public init() { }
+    
+    public private(set) var cloneInCallCount = 0
+    public private(set) var cloneInParameters: [(url: String, path: AbsolutePath)] = []
+    public func clone(url: String, in path: AbsolutePath) throws {
```

Comment:
I personally think the approach of using a closure here is usable in more contexts and it might be cumbersome, in some cases, to use it like this

by closure I mean stubbing the whole method in the following way:
```swift
public var cloneStub: ((String, AbsolutePath) throws -> ())?
```

But I know that this style is already in the codebase, so stick to whatever *you* prefer 🙂